cypress/e2e/MarkdownOrganizeLinks.spec.ts

```diff
@@ -1,16 +1,18 @@
+import { getIframe } from '../fixtures/utils';
+
 describe('Markdown Editor / Organize Links', () => {
   const selectors = {
     getInput: () => {
-      return cy.get('[data-test-id="markdown-textarea"] [contenteditable]');
+      return getIframe().findByTestId('markdown-textarea').find('.CodeMirror-code');
```

Comment:
on the other ones we still use contenteditable, whats the difference here for .CodeMirror-code?

Reply:
Honestly not sure at this point, it was a long time since this change. I think they both should work. I vaguely remember having some problems with HTML attributes

Reply:
the test fails when we use contenteditable, the switch to code mirror code was the fix